sdk/communication/Azure.Communication.MediaComposition/tests/MediaCompositionClient/MediaCompositionClientLiveTests.cs

```diff
@@ -0,0 +1,391 @@
+                       call: "teamsMeeting")
+                {
+                    PlaceholderImageUri = "https://imageendpoint"
+                }
+            },
+            ["support"] = new()
+            {
+                Participant = new(
+                       id: new() { MicrosoftTeamsUser = new("fa4337b5-f13a-41c5-a34f-f2aa46699b61") },
+                       call: "teamsMeeting")
+                {
+                    PlaceholderImageUri = "https://imageendpoint"
+                }
+            },
+            ["teamsMeeting"] = new()
+            {
+                TeamsMeeting = new("https://teamsJoinUrl")
+            }
+        };
+
+        public MediaCompositionClientLiveTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        [Test]
+        [TestCase(AuthMethod.ConnectionString, TestName = "CreateMediaCompositionWithConnectionString")]
+        [TestCase(AuthMethod.KeyCredential, TestName = "CreateMediaCompositionWithKeyCredential")]
+        // TODO: AAD authentication for tests not supported in INT environment. Uncomment the line below once we have ppe/prod deployment
+        // [TestCase(AuthMethod.TokenCredential, TestName = "CreateMediaCompositionWithTokenCredential")]
+        public async Task CreatePresenterMediaComposition(AuthMethod authMethod)
+        {
+            var mediaCompositionClient = CreateClient(authMethod);
+            var response = await CreateMediaCompositionHelper(mediaCompositionClient);
+            Assert.IsNotNull(response.Value.Layout.Presenter);
+            Assert.IsNull(response.Value.Layout.Grid);
+            Assert.AreEqual(response.Value.Inputs.Count, 3);
+            Assert.AreEqual(response.Value.Outputs.Count, 1);
+            Assert.AreEqual(response.Value.StreamState, CompositionStreamState.NotStarted);
+            await mediaCompositionClient.DeleteAsync(mediaCompositionId);
+        }
+
+        [Test]
+        public async Task CreateMediaCompositionNullInputsShouldThrow()
+        {
+            try
+            {
+                var mediaCompositionClient = CreateClient();
+                var layout = new MediaCompositionLayout()
+                {
+                    Resolution = new(1920, 1080),
+                    Presenter = new("presenter", "support")
+                };
+
+                var outputs = new Dictionary<string, MediaOutput>()
+                {
+                    {
+                        "acsGroupCall",
+                        new()
+                        {
+                            GroupCall = new("d12d2277-ffec-4e22-9979-8c0d8c13d193")
+                        }
+                    }
+                };
+                await mediaCompositionClient.CreateAsync(mediaCompositionId, layout, null, outputs);
+            }
+            catch (RequestFailedException ex)
+            {
+                Assert.NotNull(ex.Message);
+                Assert.True(ex.Message.Contains("400"));
```

Comment:
Should we check the Message or rather the Status here? I would imagine that the message will change to something more specific in the future.

Reply:
Great suggestion! Will update in my next commit